pkg/inventory/inventory-client-factory.go

```diff
@@ -8,19 +8,20 @@ import (
 )
 
 var (
-	_ ClientFactory = ClusterClientFactory{}
```

Comment:
Wanna change the file name too? configmap-client-factory.go perhaps?

Reply:
done